plugins/woocommerce/tests/e2e-pw/global-teardown.js

```diff
@@ -1,10 +1,17 @@
 const { chromium } = require( '@playwright/test' );
+const { ADMIN_USER, ADMIN_PASSWORD } = process.env;
+const adminUsername = ADMIN_USER ?? 'admin';
+const adminPassword = ADMIN_PASSWORD ?? 'password';
```

Comment:
Same thing I did on the global setup: I replaced the hard-coded login credentials with variables that could be overridden.